src/test/java/study/LadderGameTest.java

```diff
@@ -30,9 +32,12 @@ void createLadder() {
     void play() {
         LadderGame ladderGame = new LadderGame(List.of(new PlayerName("a"), new PlayerName("b")), 3);
         Ladder ladder = new Ladder(List.of(
-                new Line(List.of(true)),
-                new Line(List.of(false)),
-                new Line(List.of(true))
+                new Line(List.of(new Point(0, Edge.first(true))
+                        , new Point(1, Edge.first(true).last()))),
+                new Line(List.of(new Point(0, Edge.first(false))
+                        , new Point(1, Edge.first(false).last()))),
+                new Line(List.of(new Point(0, Edge.first(true))
+                        , new Point(1, Edge.first(true).last())))
```

Comment:
Nit: 각 `Point`를 쉽게 생성할 수 있는 메서드를 제공해보는 것도 좋을 것 같습니다.
```suggestion
            new Line(List.of(right(0), left(1))),
            new Line(List.of(pass(0), pass(1))),
            new Line(List.of(right(0), left(1)))
```